### PR TITLE
Update deployment, statefulset, daemonset in project user guide

### PR DESCRIPTION
**Pull request by rayzhou2017**
Signed-off-by: Ray Zhou <ray@yunify.com>

### CODE REVIEW

File: content/en/docs/project-user-guide/application-workloads/daemonsets.md [hunk 1]
```diff
@@ -21,13 +21,14 @@ DaemonSets are very helpful in cases where you want to deploy ongoing background
...
-- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [the steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
 
 ## Create a DaemonSet
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **DaemonSets** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **DaemonSets** and click **Create**.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/application-workloads/daemonsets.md:31]: Users cannot choose **DaemonSets** directly if they go to **Application Workloads**. The tab is in **Workloads**. If you want to add **Application Workloads** here, try this:

Go to **Application Workloads** of a project, select **Workloads**,  and click **Create** under the tab **DaemonSets**.

File: content/en/docs/project-user-guide/application-workloads/deployments.md [hunk 2]
```diff
@@ -13,13 +13,14 @@ For more information, see the [official documentation of Kubernetes](https://kub
...
-- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [the steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
 
 ## Create a Deployment
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **Deployments** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **Deployments** and click **Create**.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/application-workloads/deployments.md:23]: Same here. Please change it to:

Go to **Application Workloads** of a project, select **Workloads**, and click **Create** under the tab **Deployments**.

File: content/en/docs/project-user-guide/application-workloads/statefulsets.md [hunk 3]
```diff
@@ -21,18 +19,21 @@ StatefulSets are valuable for applications that require one or more of the follo
...
 
 ## Create a StatefulSet
 
-In KubeSphere, a **Headless** service is also created when you create a StatefulSet. You can find the headless service in **Services** under **Application Workloads** in a project.
+In KubeSphere, a **Headless** service is also created when you create a StatefulSet. You can find the headless service in [Services](../services/) under **Application Workloads** in a project.
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **StatefulSets** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **StatefulSets** and click **Create**.
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/application-workloads/statefulsets.md:36]: Same here. Please change it to:

Go to **Application Workloads** of a project, select **Workloads**, and click **Create** under the tab **StatefulSets**.

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments. Others look good to me. Thanks.

File: content/en/docs/project-user-guide/application-workloads/daemonsets.md (hunk 1 above)
Comment (rayzhou2017) [reply content/en/docs/project-user-guide/application-workloads/daemonsets.md:31]: I see, I will update it in a different PR.

**Pull request merged**